base/src/main/java/io/spine/validate/ExternalConstraints.java

```diff
@@ -80,6 +81,30 @@ private ExternalConstraints(ImmutableSet<ExternalMessageConstraint> constraints)
         return Holder.instance.constraints;
     }
 
+    /**
+     * Checks if there is in external constraint targeting the field with the given name in
+     * the given type.
+     *
+     * <p>Example. If an external constraint is declared as follows:
+     * {@code (constraint_for) = "foo.example.Customer.name"}, a call to this method with params
+     * {@code definedFor(Customer.getDescriptor(), "name")} will return {@code true}.
+     *
+     * @param containerType
+     *         the type which contains the target field
+     * @param fieldName
+     *         the target field
+     * @return {@code true} if there is an external constraint targeting the field,
+     *         {@code false} otherwise
+     */
+    public static boolean definedFor(Descriptor containerType, String fieldName) {
+        for (ExternalMessageConstraint constraint : all()) {
```

Comment:
This one looks `anyMatch()`.

Reply:
As discussed, we use a loop here due to performance considerations.